substrate/runtime-support/src/storage/mod.rs

```diff
@@ -122,58 +122,77 @@ pub fn put_raw(key: &[u8], value: &[u8]) {
 	runtime_io::set_storage(&twox_128(key)[..], value)
 }
 
-struct RuntimeStorage;
+pub struct RuntimeStorage;
```

Comment:
doe it need to be pub? should have docs if so

Reply:
yup it's needed for the macro